### PR TITLE
Implement random xkcd comics #61

### LINKED ISSUE #58: Random xkcd comics
As the professor, I would like to be able to get random xkcd comics in order to amuse myself.

### LINKED ISSUE #61: Implement random xkcd comics
Implement User Story #58.

Completion criteria: Pull request containing a command that generates random xkcd comics.

Due: Tuesday, April 20

### PR DESCRIPTION
**Pull request by sethsarchet**

### CODE REVIEW

File: cogs/fun.py [hunk 1]
```diff
@@ -312,5 +312,13 @@ async def jokes(self, ctx):
         ]
         await ctx.send(random.choice(allJokes))
 
+    ## @story{58} The bot returns a random xkcd comic between 1 and 2400
+    @commands.command(aliases=['comic'])
+    async def xkcdcomic(self, ctx):
+        """Returns a random xkcd comic"""
+
+        await ctx.send(f"https://xkcd.com/" + str(random.randint(1,2400)) + "/")
```
Comment (jluckenbill) [inline cogs/fun.py:320]: Good that you used HTTPS here

**Review by jluckenbill — APPROVED** [review]
Code looks good. I'm approving your request.

Your commit history demonstrates that you used what I call incremental test driven development (this is my own definition). That is the same development style I use and I think it is a good way of doing things. The only advice I would give you on this is that when it comes to version control, you don't need to commit every single change you make during this process. You can if you want to but you should recognize that if you were working on a larger codebase with more significant additions, your commit log will become very long and likely more difficult to read through.

File: cogs/fun.py [hunk 2]
```diff
@@ -312,5 +312,13 @@ async def jokes(self, ctx):
         ]
         await ctx.send(random.choice(allJokes))
 
+    ## @story{58} The bot returns a random xkcd comic between 1 and 2400
```
Comment (NateDawg96) [inline cogs/fun.py:315]: Good commenting. I am curious about the wording of "returns a random xkcd comic between 1 and 2400." Is that the number of comics available from the source site you're using?

**Review by NateDawg96 — APPROVED** [review]

**Review by dward118 — APPROVED** [review]
I like how the bot has a number of options to select from randomly. It would have been nice to add something to where if a comic has been shown to a user in a certain time frame after running a command then it will show a different comic since it does technically have a chance to return the same comic even though it is a low chance. However, nice job.

File: cogs/fun.py (hunk 2 above)
Comment (jcgoble3) [reply cogs/fun.py:315]: xkcd posts new comics three times a week on Monday, Wednesday, and Friday. Yesterday's comic is number 2455. So the limit would have to be periodically increased, though not frequently (100 new comics per 33 1/3 weeks).

**Pull request merged**